tern/report/errors.py

```diff
@@ -81,6 +81,7 @@
     '''registry.\n'''
 empty_layer = '''Empty layer. Nothing to do.\n'''
 no_layer_created_by = "No created_by information for layer."
+no_able_to_analyze = "unknown {entity}. please analyze separately "
```

Comment:
small nit: `not_able_to_analyze` and capitalize first words of each sentence + add period: `Unknown {entity}. Please analyze separately."